src/main/Validator.java

```diff
@@ -0,0 +1,82 @@
+package main;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Validator {
+
+	private List<BankAccount> accounts = new ArrayList<>();
+
+	public boolean checkValidAccountType(final String type) {
+		List<String> validAccounts = Arrays.asList("checking", "savings", "cd");
+
+		if (validAccounts.contains(type)) {
```

Comment:
no if needed. can reduce you code to just
`return validAccounts.contains(type)`